spec/integration/client_side_encryption/queryable_encryption_examples_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe 'Queryable encryption examples' do
+  require_libmongocrypt
+  min_server_version '6.0.0-rc8'
+  require_topology :replica_set, :sharded, :load_balanced
+
+  include_context 'define shared FLE helpers'
+
+  it 'uses queryable encryption' do
+    #  Drop data from prior test runs.
+    authorized_client.use('docs_examples').database.drop
+    authorized_client.use(key_vault_db)[key_vault_coll].drop
```

Comment:
`key_vault_db_name`, since we do have a `Database` object that this could be confused with.

Reply:
And probably `key_vault_coll_name` too.